packages/prosekit/CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # prosekit
 
+## 0.22.0-beta.21
+
+### Patch Changes
+
+- [`e9ab4f4`](https://github.com/ocavue/prosekit/commit/e9ab4f4e23b631660db99e064b7e1102364cac1a) ![](https://prosekit.dev/b/extensions)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)